src/App.jsx

```diff
@@ -1,22 +1,16 @@
 import { Component } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, Route } from 'react-router-dom';
 import './App.css';
-import ListSuites from './components/list_suites';
-
-const DEFAULT_BENCHMARKS = [
-  'ARES6',
-  'JetStream',
-  'motionmark_htmlsuite',
-  'motionmark_animometer',
-];
+import ListSuites from './components/listSuites';
+import PlotSuite from './components/plotSuite';
 
 export default class App extends Component {
   state = {
     hasError: undefined,
     configuration: {
-      linux64: DEFAULT_BENCHMARKS,
-      'windows7-32': DEFAULT_BENCHMARKS,
-      'windows10-64': DEFAULT_BENCHMARKS,
+      linux64: ['ARES6', 'JetStream', 'dromaeo_dom', 'motionmark_htmlsuite', 'motionmark_animometer'],
```

Comment:
Any reason you didn't just make `linux64: DEFAULT_BENCHMARKS + ['dromaeo_dom']`, instead of duplicating the definitions here?

Reply:
Good catch!